src/battleground/BattlegroundContainer.js

```diff
@@ -0,0 +1,61 @@
+//@flow strict
+
+import * as React from 'react';
+import Battleground from './Battleground.js';
+import HealthBarsField from './HealthBarsField.js';
+import Navbar from '../globalComponents/Navbar.js';
+import Tank from '../tanks/Tank.js';
+import "../Main.css";
+
+type Props = {||};
+
+type State = {
+	playerOneTank?: Tank,
```

Comment:
These are optional here, but in HealthBarsField they are non-optional. Remove the question mark from all of them, or make sure they all have question marks.